aleat3/applications/__init__.py

```diff
@@ -20,7 +20,7 @@
                     diceMove)
 
 class Transfer:
-    def __init__(self):
+    def __init__(self) -> None:

```

Comment:
Side note: When something is returning nothing (like `__init__()` does), it must be annotated as `-> None`:

```python
def someFunction() -> None:
    pass
```

Reply:
I think that annotations are not so important as well, so this PR can be considered as trivial, isn't it?